tools/src/main/scala/scala/scalanative/build/Build.scala

```diff
@@ -172,6 +178,12 @@ object Build {
     ) {
       LLVM.link(config, linkerResult, compiled)
     }
+
+    def dsymutil(linked: Path): Unit = time(
+      s"Running dsymutil on ${linked.getFileName}"
```

Comment:
We should not require users to now what dsymutil is and what it does. Maybe let's just introduce a single logging of time spent in whole postprocessing stage (based on suggestion to move all postprocessing to seperate step). At that point can remove this method and use `LLVM.dsymutil` directly in postprocessing stage.